### PR TITLE
Set scrollbar colors as global styles

### LINKED ISSUE #1337: Theme-based scrollbar background colors and sizes are hardcoded in SCSS
If an extension needs scrollbars, there is currently no way for it to get the proper colors and sizes because they are hardcoded in SCSS, see https://github.com/lensapp/lens/blob/70a8982c9f6396107f92aeced465620761d90726/src/renderer/components/mixins.scss#L25-L32

```
@mixin custom-scrollbar($theme: light, $size: 7px, $borderSpacing: 5px) {
  $themes: (
    light: #5f6064,
    dark: #bbb,
  );

  $scrollBarColor: if(map_has_key($themes, $theme), map_get($themes, $theme), none);
  $scrollBarSize: calc(#{$size} + #{$borderSpacing} * 2);
```

The light and dark colors, as well as default size and border width, should be provided as themed CSS Variables:

```css
--scrollbarBackground: <color>;
--scrollbarSize: 7px;
--scrollbarBorderWidth: 5px;
```

Please remember to add these to the [color reference](https://github.com/lensapp/lens/blob/master/docs/extensions/capabilities/color-reference.md) in the docs.

### LINKED ISSUE #1483: Unnecessary scrollbars in Drawer
When no events provided for workload, horizontal scrollbar appears in Details panel (Drawer).

drawer scrollbars

### PR DESCRIPTION
**Pull request by aleksfront**
This PR simplifies using themable scrollbar colors and eliminates of using sass `custom-scrollbar` mixin.

From now extensions will also use custom scrollbars defined in global styles for sake of consisten view across all Lens components.

chart details
sidebar
logs

Fixes #1337 
Fixes #1483 
Part of #1298

### CODE REVIEW

File: src/renderer/components/+preferences/preferences.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
     .Badge {
       display: flex;
       margin: 0;
-      margin-bottom: 1px;
+      margin-bottom: 1px!important;
```
Comment (Nokel81) [inline src/renderer/components/+preferences/preferences.scss]: Is this really necessary?

File: src/renderer/components/wizard/wizard.scss [hunk 2]
```diff
@@ -15,10 +15,14 @@
...
 
   @mixin scrollableContent() {
-    @include custom-scrollbar($theme: dark);
+    overflow: auto;
     padding: var(--wizard-spacing);
     height: var(--wizard-content-height);
     max-height: var(--wizard-content-max-height);
+
+    &::-webkit-scrollbar-thumb {
+      background-color: #bbbbbb; // Always light scrollbar in dialogs
```
Comment (Nokel81) [inline src/renderer/components/wizard/wizard.scss]: Do we actually want to force this or should we make it a variable like all other colours?

File: src/renderer/components/+preferences/preferences.scss (hunk 1 above)
Comment (aleksfront) [reply src/renderer/components/+preferences/preferences.scss]: Not really. Fixed.

File: src/renderer/components/wizard/wizard.scss (hunk 2 above)
Comment (aleksfront) [reply src/renderer/components/wizard/wizard.scss]: Removed these lines.

File: src/renderer/components/virtual-list/virtual-list.scss [hunk 3]
```diff
@@ -2,12 +2,6 @@
...
 
   > .list {
-    @include custom-scrollbar;
-
-    .theme-light & {
-      @include custom-scrollbar(dark);
-    }
-
-    overflow-y: overlay !important;
+    overflow-y: overlay!important;
```
Comment (ixrock) [inline src/renderer/components/virtual-list/virtual-list.scss:5]: Why `!important`?

**Review by ixrock — APPROVED** [review]
LGTM

File: src/renderer/components/virtual-list/virtual-list.scss (hunk 3 above)
Comment (aleksfront) [reply src/renderer/components/virtual-list/virtual-list.scss:5]: It overrides `overflow: auto` inline styles defined by `VirtualList` component.

Comment (ixrock) [reply src/renderer/components/virtual-list/virtual-list.scss:5]: Or by `react-window` internals did you mean?

**Comment by stefcameron** [comment]
Thank you!

**Pull request merged**